web-ng/src/app/shared/models/project.model.ts

```diff
@@ -15,14 +15,27 @@
  */
 
 import { Layer } from './layer.model';
-import { List } from 'immutable';
+import { Form } from './form/form.model';
 import { StringMap } from './string-map.model';
+import { Map } from 'immutable';
 
 export class Project {
   constructor(
     readonly id: string,
     readonly title: StringMap,
     readonly description: StringMap,
-    readonly layers: List<Layer>
+    readonly layers: Map<string, Layer>
   ) {}
+
+  getForm(layerId: string,  formId: string): Form {
```

Comment:
Nit: Extra space here, missing spaces after "if", use single instead of double quotes.

`npm run fix` should fix these.